darylldoyle/safe-svg
# Fixed fill-rule browser error

### Description of the Change
Fixed the browser error for svg icon that was caused by the `fill-rule` attribute.
Closes #149 

### How to test the Change
=> In `wp-config.php` file, enable script debug: `define( 'SCRIPT_DEBUG', true );`
=> Create a new page and add the Inline SVG block from block inserter
=> Add any svg image and the browser warning should go away.

### Changelog Entry
> Fixed - SVG Icon Browser Warning


### Credits
Props @zamanq @jeffpaul 


### Checklist:
